main/index.html

```diff
@@ -144,7 +151,8 @@ <h3 class="timeline__title">Repellendus nostrum alias odit.</h2>
                         </div>
                     </div>
                 </section>
-                <section id="faq" class="fullpage">
```

Comment:
@Anupam-dagar isnt this the class you added after my request ?

Reply:
yeah, it makes the faq section a fullpage one.

Reply:
It was fitting my screen height so i didnt notice that. Ill do something else for this then.
fullpage class being used is same as the one being used in fullpage.js?

Reply:
Change the screen size and see . Always . Being responsive is our number one priority

Reply:
No fullpage js is not linked in the website, fullpage class has this definition
.fullpage{
height: 90vh;
}

Reply:
Please revert this back. This was not the meaning of the issue